### PR TITLE
Initial implementation draft

### PR DESCRIPTION
**Pull request by jcechace**

### CODE REVIEW

File: bin/console [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
```
Comment (pestanko) [inline bin/console:1]: File can be deleted

File: bin/setup [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```
Comment (pestanko) [inline bin/setup:1]: File can be deleted

File: lib/rb_shift/client.rb [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+module RbShift
+  class Client
+    attr_reader :token, :url
+
+    def initialize(url, bearer_token)
+      @token      = bearer_token
+      @url        = url
+      @kubernetes = RestClient::Resource.new "#{url}/api/v1",
+                                             verify_ssl: OpenSSL::SSL::VERIFY_NONE,
```
Comment (pestanko) [inline lib/rb_shift/client.rb]: This should be parametrized

File: bin/console (hunk 1 above)
Comment (jcechace) [reply bin/console:1]: Sure.. on the other hand I use it for testing purposes -- it starts ruby repl and loads the module right away.

File: lib/rb_shift/client.rb (hunk 3 above)
Comment (jcechace) [reply lib/rb_shift/client.rb]: Agreed, SSL validation should be a subject to configuration.

File: bin/console (hunk 1 above)
Comment (pestanko) [inline bin/console]: This file is not required I think.

File: bin/setup (hunk 2 above)
Comment (pestanko) [inline bin/setup]: Neither this one.

**Comment by jcechace** [comment]
waiting for #2

**Pull request merged**